2023/10/25/main.go

```diff
@@ -0,0 +1,17 @@
+// Daily Joel 2023-10-25
+//
+// Pointers.
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	name := ""
+
+	p := &name
+	*p = "Joel"
```

Comment:
Assigning the variable p with where name is in the memory. Then assigning, by pointing at what p's referencing to with "Joel".